landscape.yml

```diff
@@ -1328,12 +1328,17 @@ landscape:
             logo: smartos.svg
             crunchbase: https://www.crunchbase.com/organization/joyent
           - item:
-            joined: '2021-02-01'
-            name: SSVM
-            homepage_url: https://www.secondstate.io/ssvm
+            extra:
+              accepted: '2021-04-28'
+              dev_stats_url: https://wasmedge.teststats.cncf.io/
```

Comment:
You can replace the `test` DevStats instance (https://wasmedge.teststats.cncf.io/) with a `prod` one (https://wasmedge.devstats.cncf.io/)

Reply:
> You can replace the `test` DevStats instance (https://wasmedge.teststats.cncf.io/) with a `prod` one (https://wasmedge.devstats.cncf.io/)

Will do!